templates/bridgeserver2.yaml

```diff
@@ -183,7 +183,7 @@ Resources:
     Type: 'AWS::ElasticBeanstalk::ConfigurationTemplate'
     Properties:
       ApplicationName: !ImportValue us-east-1-bridgeserver2-common-BeanstalkAppName
-      SolutionStackName: '64bit Amazon Linux 2 v4.2.11 running Tomcat 8.5 Corretto 8'
+      SolutionStackName: '64bit Amazon Linux 2 v4.7.0 running Tomcat 9 Corretto 8'
```

Comment:
hopefully this goes smoothly but from my experience it will require some hand holding.

Reply:
yeah, according to the changelog it doesn't seem like it'll make a difference for us, but I'm about to find out 🤞🏼